Sources/Plasma/PubUtilLib/plModifier/plResponderModifier.cpp

```diff
@@ -790,16 +789,12 @@ void plResponderModifier::ILog(uint32_t color, const char* format, ...)
     va_end(args);
 
     // Strip the redundant part off the key name
-    char logLine[512];
-    const char* modPos = strstr("_ResponderModifier", keyName);
```

Comment:
Deledrius said something about searching for the haystack inside the needle :trollface: